Platforms/QemuQ35Pkg/Plugins/QemuRunner/QemuRunner.py

```diff
@@ -128,8 +128,8 @@ def Runner(env):
 
         # Add XHCI USB controller and mouse
         args += " -device qemu-xhci,id=usb"
-        args += " -device usb-mouse,id=input0,bus=usb.0,port=1"  # add a usb mouse
-        #args += " -device usb-kbd,id=input1,bus=usb.0,port=2"    # add a usb keyboar
+        args += " -device usb-tablet,id=input0,bus=usb.0,port=1"  # add a usb mouse
+        #args += " -device usb-kbd,id=input1,bus=usb.0,port=2"    # add a usb keyboard

```

Comment:
Do you know of any reason why the kb is not included here?

@cfernald Do you remember by any chance?

Reply:
It's been commented out since at least this commit in April 2021 - https://github.com/microsoft/mu_tiano_platforms/commit/4a343a57975c8cff0b811a8188a2e4bf40625bff

Reply:
I enabled it and could see it show up as a device handle in EFI shell with keyboard working in EFI shell and the OS. Though I'm not sure I want to enable this right now unless we have a specific reason to do so. The `usb-tablet` device change improves some inconsistency I saw using kvm (may not be related to kvm) and definitely tracks the cursor better.

Reply:
Will go ahead and merge for now.